modules/system/db.go

```diff
@@ -33,5 +32,15 @@ func (f *DBStore) Set(ctx context.Context, item StateItem) error {
 	if err != nil {
 		return err
 	}
-	return system.SaveAppStateContent(ctx, item.Name(), util.BytesToReadOnlyString(b))
+	return system.SaveAppStateContent(ctx, item.Name(), unsafeBytesToString(b))
+}
+
+// unsafeBytesToString is copied from modules/util/string.go to avoid import cycle.
```

Comment:
`modules/util` shouldn't have dependency on `modules/system` package.

So either: 

* Fix the dependency
* Keep using old `goldmark/util`

Reply:
Agree, fixed.

(Something is broken with my IDE)